zap/src/main/resources/org/zaproxy/zap/resources/drivers.xml

```diff
@@ -60,6 +60,12 @@
 		<slot>1</slot>
 		<slotListIndex>0</slotListIndex>
 	</driver>
+	<driver>
+		<name>AWP Identity Client - Linux</name>
+		<path>/usr/local/AWP/lib/libOcsPKCS11Wrapper.so</path>
+		<slot>0</slot>
+		<slotListIndex>0</slotListIndex>
+	</driver>
```

Comment:
Should be moved up to keep them in alphabetic order.

Reply:
fixed in 98c6eed2a83710bd55959dfc2e5ace154691aca4